docs/netstandard-20/packaging.md

```diff
@@ -0,0 +1,350 @@
+dependency on those. For instance, the dependency on `NETStandard.Library16`
+is specific to the `netstandard16` TFM.
+
+So when running package restore for `netstandard16`, NuGet will only download
+two packages, `NETStandard.Library` as well as `NETStandard.Library16`. When the
+developer changes the TFM to `netstandard20` it will trigger another package
+restore which will then download `NETStandard.Library20` package instead.
+
+Each of the `NETStandard.LibraryXX` packages will have a structure like this:
+
+* `ref/netstandard{version}`
+    - Contains `netstandard.dll` reference assembly
+    - Contains `System.Runtime`-based facades, type forwarding to `netstandard.dll`
+    - Contains `mscorlib`-based facades, type forwarding to `netstandard.dll`
+* `ref/{platform}`
+    - Contains the placeholder `_._` so that no other platforms gets any compile
+      time assets as they are supposed to be provided by the platform.
+    - We will omit version in the TFM (effectively using 0) to avoid having to
+      rev this package every time the platform ships a new version
+
+For `NETStandard.Library20` we'll have to add the following additional assets
+in order to provide the support for .NET Framework 4.6.1:
+
+* `ref/net461`
+    - Contains the placeholder `_._` so that no assets are inserted by NuGet for
+      .NET Framework 4.6.1
+* `tools/net461`
+    - Contains a single `netstandard.targets` file that provides reference items
+      to all the facades
+    - Contains `netstandard` facade, type forwarding to `mscorlib.dll`
+    - Contains `System.Runtime`-based facades, type forwarding to `mscorlib.dll`     
+
+### Layout of contract packages (`System.*`)
+
+Our goal is to discontinue the existing fine-grained packages we have been using
+for .NET Standard 1.x as well as for .NET Core.
+
+We don't plan to update these packages. Instead we'll rely on [MSBuild conflict
+resolution](#msbuild-conflict-resolution) for package graphs that include the
+old packages.
+
+## Layout of out-of-band packages (e.g. `System.Collections.Immutable`)
+
+Components that don't have APIs that ship with any platform (i.e. are fully
+out-of-band) will remain as individual packages. We'll update them to target
+.NET Standard so that they can be used across all .NET platforms.
+
+### Layout of PCL Compat package (`Microsoft.Portable.Compat`)
+
+Similar to the existing `System.*` packages this package will be discontinued as
+it's replaced by `NETStandard.Library`.
+
+We don't plan to update this package. Instead we'll rely on [MSBuild conflict
+resolution](#msbuild-conflict-resolution) for package graphs that include this
+package.
+
+### Layout of `Microsoft.NETCore.App`
+
+* No dependency to `NETStandard.Library` 
+* `ref/netcoreapp{vNext}`
+    - Contains `netstandard.dll` facade type forwarding to `System.Runtime`
+    - Contains `mscorlib`-based facade type forwarding to `System.Runtime`
+    - Contains `System.Runtime`-based reference assemblies
```

Comment:
Contains all of them? Or only things that intersect with netstandard? What about things that are in the implementation closure of the things that make up netstandard? (ex: System.Buffers, or System.Collections.Immutable)